### PR TITLE
Add DPT

### PR DESCRIPTION
**Pull request by d-cunningham**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mxr-conda** [comment]
Agree to maintain

File: recipes/dpt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  - url: https://github.com/isl-org/DPT/releases/download/1_0/dpt_large-ade20k-b12dca68.pt
+    sha256: b12dca688686e8af6758a32b437368c0a6ebf5fde9f8ab7819e0dd9dab8eda22
+    folder: weights
+  - url: https://github.com/isl-org/DPT/releases/download/1_0/dpt_large-midas-2f21e586.pt
+    sha256: 2f21e586477d90cb9624c7eef5df7891edca49a1c4795ee2cb631fd4daa6ca69
+    folder: weights
+  
+build:
+  number: 0
+  skip: true  # [win]
```
Comment (dopplershift) [inline recipes/dpt/meta.yaml:35]: This looks like it's pure Python and could be noarch (we can ignore the Windows CI failure due to missing pytorch):
```suggestion
  noarch: python
```

File: recipes/dpt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (dopplershift) [inline recipes/dpt/meta.yaml:40]: For noarch:
```suggestion
    - python >=3.7
```

File: recipes/dpt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    - pip
+  run:
+    - python
```
Comment (dopplershift) [inline recipes/dpt/meta.yaml:43]: For noarch:
```suggestion
    - python >=3.7
```

**Review by dopplershift — CHANGES_REQUESTED** [review]

File: recipes/dpt/meta.yaml (hunk 1 above)
Comment (d-cunningham) [reply recipes/dpt/meta.yaml]: Awesome changes made, I didn't know if the windows failure could be ignore, so I skipped win. Thanks for the review!

Comment (dopplershift) [reply recipes/dpt/meta.yaml]: Noarch packages are only (currently at least) built on Linux anyways, so Windows users trying to install it just get a missing dependency error. In exchange, the build resources needed for this package (and the amount of rebuilds) is much much less.

File: recipes/dpt/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    - pip
+  run:
+    - python >=3.7
+    - pytorch
+    - py-opencv
+    - torchvision
+    - timm
+
+test:
+  requires:
```
Comment (dopplershift) [inline recipes/dpt/meta.yaml:50]: Can you add some import tests to validate the package? Something like:
```yaml
    imports:
      - useful.package.name
```

**Review by dopplershift — APPROVED** [review]

**Comment by dopplershift** [comment]
Ignoring noarch Windows build failure due to missing dependency.

**Pull request merged**